setup.cfg

```diff
@@ -55,6 +55,7 @@ install_requires =
     semantic-version
     tenacity
     tqdm
+    datalad
```

Comment:
Datalad is only used while testing, so it should be added to the `test` extras under `[options.extras_require]`, not to `install_requires`.  Also, when adding it, please keep the list in alphabetical order.